src/components/InputText/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './styles.module.scss';
+
+function InputText({ label, type, name, reference, errors }) {
+  return (
+    <>
+      <label className={styles.label}>{label}</label>
+      {errors && <span className={styles.error}>{errors.message}</span>}
```

Comment:
remember that you always should have just one error, so renaamee it to `error` and the propType should be string